src/library/trackset/baseplaylistfeature.cpp

```diff
@@ -600,6 +600,101 @@ void BasePlaylistFeature::slotExportPlaylist() {
     }
 }
 
+void BasePlaylistFeature::slotExportPlaylists() {
+    const PlaylistDAO::HiddenType hidden = PlaylistDAO::PLHT_NOT_HIDDEN;
+
+    QList<QPair<int, QString>> allPlaylists = m_playlistDao.getPlaylists(hidden);
+
+    QPair<int, QString> thePair;
+    int playlistId;
+    QString playlistName;
+
+    for (int i = 0; i < allPlaylists.length(); i++) {
+        thePair = allPlaylists.at(i);
+        playlistId = thePair.first;
+        playlistName = m_playlistDao.getPlaylistName(playlistId);
```

Comment:
Please use here a range based loop, because i is only used to access the allPlaylists list. Here a syntax with the pair, that removes some of the pain. 
```suggestion
    for (const auto& [playlistId, playlistName] : allPlaylists) {
```